pkg/doc/did/doc.go

```diff
@@ -777,11 +777,16 @@ type didKeyResolver struct {
 	PubKeys []PublicKey
 }
 
-func (r *didKeyResolver) Resolve(id string) (*verifier.PublicKey, error) {
+func (r *didKeyResolver) Resolve(id string) (interface{}, error) {
```

Comment:
this change __only__ makes sense if pubKey is an opaque object to every package except for a kms implementation.

but I see this package constructing a generic return value. this is confusing.

Reply:
I agree that we should not return interface here.